### ISSUE TITLE
Cydia won’t let me install tweaks 

### ISSUE THREAD

**Issue by gabigol1000** [checkra1n/BugTracker#159]
**What device + iOS version are you on?**
iPhone X IOS 13.2.2

**What checkra1n version are you using?**
0.9.1
**What are the steps to reproduce the issue?**
1.Fix this issue 
2. In the next update resolve this issue 
3.Read this issue so you guys can fix it for other users too 
...

**What do you expect, and what is happening instead?**
I expect to to able to download tweaks but except the error pops up 
![B6831766-6AAF-4D46-A726-D9F06DE0BCC4](https://user-images.githubusercontent.com/57612421/68563826-ba959680-0403-11ea-8d7d-bfe214432c64.png)

**Any other info, error logs, screenshots, ...?**Screenshot is on top. Thank you

**Comment by Lonegwadiwaitor** [comment]
Yeah this is easily fixed, please open SSH or SFTP and navigate to /etc/apt/sources.list.d and delete both cydia.list and mobilesubstrate.list, Cydia should remake these files, but back them up anyway just incase.

**Comment by nullpixel** [comment]
Did you jailbreak with unc0ver on this device previously?
